File: InsertionSort.java
public class InsertionSort {
    void sort(int arr[]) {
        for (int j = 1; j < arr.length; j++) {
            // Find the correct key
            int key = arr[j];
            int i = j - 1;

            while (i >= 0 && arr[i] > key) {
                arr[i + 1] = arr[i];

                i--;
            }

        }
    }
}

File: Sort/InsertionSort/InsertionSort.py
def InsertionSort(arr):
    for i in range(len(arr)):
        key = arr[i]
        j = i - 1

        while j >= 0 and key < arr[j]:
            arr[j + 1] = arr[j]
            j -= 1
            arr[j + 1] = key




arr = [23, 45, 3, 5, 11]
InsertionSort(arr)
for i in range(len(arr)):
    print(arr[i])